toast-er.html

```diff
@@ -261,6 +288,10 @@
         }
       },
 
+      _buttonTap: function(event) {
+        this.fire("toast-er-" + event.target.buttonEvent);
```

Comment:
@wincinderith The element name must have a dash. So, we'll follow the spec for the event name.